pkg/postgres/pool.go

```diff
@@ -0,0 +1,30 @@
+package postgres
+
+import (
+	"context"
+	"fmt"
+	"github.com/jackc/pgx/v5/pgxpool"
+)
+
+type PgConfig struct {
+	Username string
+	Password string
+	Host     string
+	Port     string
+	Database string
+}
+
+func NewPgConfig(username, password, host, port, database string) *PgConfig {
+	return &PgConfig{Username: username, Password: password, Host: host, Port: port, Database: database}
+}
+
+func NewClient(cfg *PgConfig) (*pgxpool.Pool, error) {
+	
```

Comment:
Makes sense to receive external context here so that the operation could be cancelled by Ctrl+C or something